src/arch/ArchHooks/ArchHooks_Win32Static.cpp

```diff
@@ -9,20 +11,22 @@
 #include <cstdint>
 #include <vector>
 
-// for QueryPerformanceCounter
 #include <windows.h>
 #include <mmsystem.h>
 #if defined(_MSC_VER)
 #pragma comment(lib, "winmm.lib")
 #endif
 
-static bool g_bTimerInitialized;
-
-/* QueryPerformanceCounter variables below.
- * QueryPerformanceCounter and QueryPerformanceFrequency expect
+/* QueryPerformanceCounter and QueryPerformanceFrequency expect
  * a LARGE_INTEGER, which is a union. These functions store data
- * in the QuadPart of the LARGE_INTEGER, which is a 64-bit integer. */
-LARGE_INTEGER g_liFrequency, g_liStartTime, g_liCurrentTime;
+ * in the QuadPart of the LARGE_INTEGER, which is a 64-bit integer.
+ * The QuadPart is accessed directly to avoid the overhead of
+ * converting the LARGE_INTEGER to a 64-bit integer.
```

Comment:
I decided this comment will probably only confuse readers so I'll remove this sentence. I'll leave the rest.